RevenueCat/Scripts/Purchases.cs

```diff
@@ -662,7 +737,12 @@ private void _receiveProducts(string productsJson)
         }
         else
         {
-            ProductsCallback(response.products, null);
+            var products = new List<Product>();
+            foreach (var productResponse in response.products)
+            {
+                products.Add(new Product(productResponse));
+            }
```

Comment:
I think you could replace these lines with
```c#
var products = response.products.Select(productResponse => new Product(productResponse))
```
https://blog.submain.com/csharp-functional-programming/

Reply:
I saw that, but I was scared to change it just in case it's not compatible with all runtimes. I will double check

Reply:
https://forum.unity.com/threads/is-c-linq-fully-reliable-supported.379326/ I am not sure